lib/utils/create-from-local-template.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const copyDirContentsSync = require('./fs/copyDirContentsSync');
+const untildify = require('untildify');
+const { renameService } = require('./renameService');
+
+const ServerlessError = require('../serverless-error');
+
+module.exports = ({ templatePath, projectDir, projectName }) => {
+  const sourcePath = untildify(templatePath);
+
+  try {
+    copyDirContentsSync(sourcePath, projectDir, { noLinks: true });
```

Comment:
Let's use `fse.copy` instead (we also consider `*Sync` functions as deprecated, and they're scheduled to be replaced, and such utils removed -> https://github.com/serverless/serverless/issues/8604)